lib/checks/label/label-content-name-mismatch.js

```diff
@@ -0,0 +1,34 @@
+function getAriaText(elm) {
+	if (elm.getAttribute('aria-label')) {
+		return elm
+			.getAttribute('aria-label')
+			.toLowerCase()
+			.trim();
+	}
+	if (elm.getAttribute('aria-labelledby')) {
```

Comment:
aria-labelledby is an element reference. You're just picking up the ID of the element, not the actual text. We have some code for this, but its not isolated properly. Let me complete my accessible name update before completing this. One of the things I'm doing there is to create a getAriaLabelledbyName function.